### ISSUE TITLE
Sync formal title

### ISSUE THREAD

**Issue by eptbertram** [systopia/de.systopia.twingle#81]
I noticed that formal_title does not sync from Twingle to my CiviCRM 5.67.2 on Drupal 10.2.2. No big deal, but strange: Twingle uses this field as optional.

**Comment by jensschuppe** [comment]
The field `formal_title` is part of the contact data sent to *XCM* when the *Twingle* field `user_title` is not empty. Maybe your *XCM* profile does not include that field in the "Fill fields" section for existing contacts?

**Comment by eptbertram** [comment]
No, Twingle field `user_title` is activated and XCM profile includes `formal_title`, too. But there is no data imported into CiviCRM. Do you have any idea why? 

![Twingle aktivierte Kontaktdatenfelder](https://github.com/systopia/de.systopia.twingle/assets/100374351/df6db5b3-572b-4779-bdde-6221818d1872)
![image](https://github.com/systopia/de.systopia.twingle/assets/100374351/e3a9a117-2635-46a5-b8b8-5e9ce813a70c)

CiviCRM 5.69.2 with Drupal 10, Twingle extension 1.3 and XCM 1.12.

**Comment by jensschuppe** [comment]
Not really … you could try to log requests to the `Contact.getorcreate` API action with e.g. https://lab.civicrm.org/extensions/logapirequests to see whether the value is included in what XCM gets passed, and also to the `TwingleDonation.submit` action to see whether twingle are sending a value in the first place.
